cpu/stm32_common/periph/eeprom.c

```diff
@@ -36,6 +37,61 @@ extern void _wait_for_pending_operations(void);
 #error "periph/eeprom: EEPROM_START_ADDR is not defined"
 #endif
 
+#if defined(CPU_MODEL_STM32L151CB)
+#define ALIGN_MASK          (0x00000003)
+#define BYTE_MASK           (0xFF)
+#define BYTE_BITS           (0x08)
+
+static void _erase_word(uint32_t addr)
+{
+    assert(addr <= EEPROM_SIZE + EEPROM_START_ADDR);
+
+    /* Wait for last operation to be completed */
+    _wait_for_pending_operations();
+
+    /* Write "00000000h" to valid address in the data memory" */
+    *(__IO uint32_t *)addr = 0x00000000;
+}
+
+static void _write_word(uint32_t addr, uint32_t data)
+{
+    assert(addr <= EEPROM_SIZE + EEPROM_START_ADDR);
+
+    /* Wait for last operation to be completed */
+    _wait_for_pending_operations();
+
+    *(__IO uint32_t *)addr = data;
+}
+#endif
+
+static void _write_byte(uint32_t addr, uint8_t data)
+{
+    /* Check the parameters */
+    assert(addr <= EEPROM_SIZE + EEPROM_START_ADDR);
```

Comment:
This seems unrelated and means the assert is checked for each byte written. I would drop this.